### PR TITLE
Split channels to send to master (stats vs. locust things)

### PR DESCRIPTION
**Pull request by joshprzybyszewski-wf**
After about 20 minutes of running, I see boomer stop sending heartbeats to the master node. I want to split out the channels we use to see if that has an affect on heartbeats.

The next step is to put the `sendMessage` in its own goroutine so that we're aren't waiting for it

### CODE REVIEW

File: client_czmq.go [hunk 1]
```diff
@@ -94,13 +96,22 @@ func (c *czmqSocketClient) sendChannel() chan *message {
...
+}
+
 func (c *czmqSocketClient) send() {
 	for {
 		select {
 		case <-c.shutdownChan:
 			return
 		case msg := <-c.toMaster:
-			c.sendMessage(msg)
+			go c.sendMessage(msg)
```
Comment (joshprzybyszewski-wf) [inline client_czmq.go]: It might just be good enough to use a goroutine to send the message -- it's likely the `serialize` call that's taking a long time

**Comment by joshprzybyszewski-wf** [comment]
@myzhan this should be ready for review

File: client_czmq.go (hunk 1 above)
Comment (myzhan) [reply client_czmq.go:109]: We need to make sure the zeromq socket is thread safe. Have you tested the serialization of big message?

**Comment by myzhan** [comment]
No progress.

**Pull request closed without merging**